tutorials/Python/Hyperspectral/hdf5-for-hyperspectral-data/Python-HDF5-BasicScript/Python-HDF5-BasicScript.md

```diff
@@ -70,31 +70,45 @@ Please note that temperature data are distributed as a flat .csv file and not as
 HDF5 file. NEON data products including eddy covariance data and remote sensing 
 data are however released in the HDF5 format.
 
+## Set working directory
+After downloading these datasets, ensure to save them to one folder to simplify importing them later. Let's set your working directory to ensure it is correct.
+
+```python
+import os
+wd=os.chdir('/yourfilepathhere') #change the file path to your working directory
+wd=os.getcwd() #request what is the current working directory
+print(wd) #show what is the current working directory
+```
 
 ## Python Code to Open HDF5 files
 
 The code below is starter code to create an H5 file in Python.
-
+```python
     if __name__ == '__main__':
 		# import required libraries
 		import h5py as h5
 		import numpy as np
 		import matplotlib.pyplot as plt
     
 		# Read H5 file
-		f = h5.File("NEON-DS-Imaging-Spectrometer-Data.h5", "r")
+		f = h5.File("NEONDSImagingSpectrometerData.h5", "r")
 		# Get and print list of datasets within the H5 file
 		datasetNames = [n for n in f.keys()]
 		for n in datasetNames:
 			print(n)
+ ```
+ we can see that the datasets within the h5 file include on reflectance,
+fwhm (full width half max, this is the distance in nanometers between the band center and the edge of the band), map info, spatialInfo and wavelength. Let's start by extracting the reflectance dataset specifically from the H5 file.
```

Comment:
I think that FWHM is the distance from the lower edge of the band to the upper edge of the band (with the edges defined as where the response curve falls to 50% of the maximum response). So I _think_ that you description of FWHM (from the middle of the band to one edge) is only half of the actual FWHM (from one edge, through the middle, to the other edge). But I might be wrong!